### PR TITLE
Adds ability to add multiple hearts.

### PR DESCRIPTION
**Pull request by AaronVasquez**
Support passing in a count that is greater than oldCount + 1. Also, gets rid of mutating state.

### CODE REVIEW

File: FloatingHearts.js [hunk 1]
```diff
@@ -14,24 +14,29 @@ class FloatingHearts extends Component {
     height: null,
   }
 
-  addHeart(index) {
-    this.state.hearts.push({
+  createHeart(index) {
+    console.log(index)
```
Comment (ignacioola) [inline FloatingHearts.js]: You forgot this one ;)

Comment (AaronVasquez) [reply FloatingHearts.js]: Oops. I'll delete it

**Review by ignacioola — CHANGES_REQUESTED** [review]
Thanks @AaronVasquez, this looks great. Could you modify just the mentioned line?

**Review by AaronVasquez — COMMENTED** [review]
Removed :)

**Pull request merged**